docs/server/source/cloud-deployment-templates/workflow.rst

```diff
@@ -7,114 +7,102 @@ We are constantly improving them.
-public and private keys for this node.
 
-One way to generate keys for your node is using the BigchinDB Python driver.
-
-    .. code:: bash
-        
-        from bigchaindb_driver.crypto import generate_keypair
-        print(generate_keypair())
+Things Each Node Operator Must Do
+---------------------------------
 
+☐ Decide on, or find out some names of the things in your node:
 
-Step 2.2: Get the HTTPS Certificates for the BigchainDB API
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#. Name of the MongoDB instance (``mdb-instance-*``)
+#. Name of the BigchainDB instance (``bdb-instance-*``)
+#. Name of the NGINX instance (``ngx-instance-*``)
+#. Name of the MongoDB monitoring agent instance (``mdb-mon-instance-*``)
+#. Name of the MongoDB backup agent instance (``mdb-bak-instance-*``)
 
-TODO(Troy): How to get certificates from certificate providers? Out of scope?
-Suggestions here?
 
+☐ Every node in a BigchainDB cluster needs its own
+BigchainDB keypair (i.e. a public key and corresponding private key).
+You can generate a BigchainDB keypair for your node, for example,
+using the `BigchainDB Python Driver <http://docs.bigchaindb.com/projects/py-driver/en/latest/index.html>`_.
 
-Step 2.3: Get the TLS Certificates to Secure MongoDB Cluster
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If you are the admin of the federation with access to the CA, you can generate a
-server and a client certificate for providing secure authentication between the
-BigchainDB and MongoDB instances, as well as between different MongoDB
-instances in the cluster.
-
-If you are not the admin, then you might have to request the federation admin
-to sign your certificates.
-
-In total, you will need to generate 4 keys and corresponding CSRs for:
+.. code:: python
+        
+   from bigchaindb_driver.crypto import generate_keypair
+   print(generate_keypair())
 
-#. Server Certificate or Member Certificate for MongoDB
-#. Client Certificate for BigchainDB
-#. Client Certificate for MongoDB Monitoring Agent
-#. Client Certificate for MongoDB Backup Agent
 
-One way to generate server and client certificates and the corresponding CSRs
-can be found :ref:`here <Generate Cluster Member Certificate for MongoDB>`
-and :ref:`here <Generate Client Certificate for MongoDB>`.
+☐ Share your BigchaindB *public* key with all the other nodes
+in the BigchainDB cluster.
+Don't share your private key.
 
 
-Step 2.3: Get 3scale API Gateway Account credentials
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+☐ Get the BigchainDB public keys of all the other nodes in the cluster.
+That list of public keys is known as the BigchainDB "keyring."
 
-This helps you set up auth, monitoring and billing of your APIs
 
-TODO(Troy): How to setup an account?
-TODO(Krish): How to get all the credentials from the portal?
+☐ Ask the managing organization
+for the FQDN used to serve the BigchainDB APIs
+and for a copy of the associated SSL/TLS certificate.
```

Comment:
Why is this required? We do not need to share keys for any certificates ever.
The other org can map the DNS from x.com to y.com using a CNAME record, and https will pick up the certificates for y.com and all works smoothly as before.